### DISCUSSION TITLE
Can I deploy rerank model with TEI in an offline environment？

### DISCUSSION THREAD

**Discussion by kennard520** [huggingface/text-embeddings-inference#389] [Q&A]
Below is my command:
`docker run --gpus all -p 8081:80 -v /root/^Cta:/data 192.168.0.140:8082/ghcr.io/huggingface/text-embeddings-inference:1.5 --model-id BAAI/bge-reranker-v2-m3 --tokenization-workers=5 --max-concurrent-requests=10  --max-batch-tokens=8192 --max-batch-requests=10 --pooling=mean `
but always download config
`2024-08-26T01:02:32.430578Z  INFO download_new_st_config: text_embeddings_core::download: core/src/download.rs:62: Downloading `config_sentence_transformers.json``
Can I deploy in an offine env?

**Comment by leversberg** [comment]
You need an internet connection to download the model for the first time from HuggingFace. Afterwards you can run it locally and offline.
Your mounted volume path seems to be wrong.

**Reply by kennard520** [reply]
> You need an internet connection to download the model for the first time from HuggingFace. Afterwards you can run it locally and offline. Your mounted volume path seems to be wrong.

yes，I have updated it, but I found aways download some configs every start time.
